hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/S3KeyGenerator.java

```diff
@@ -54,7 +64,8 @@
   private String bucketName;
 
   @Option(names = {"-s", "--size"},
-      description = "Size of the generated key (in bytes)",
+      description = "Size of the generated key (in bytes) or size of one "
+          + "multipart upload part (in case of multipart upload)",
```

Comment:
Minor: Do you think we need some check on the size, when multipart upload option is specified, as in MPU each part should be at least 5MB(except last part) But here we use same part size having that check would be useful, otherwise we shall get an error during complete MPU afteer uploading 10 parts.

Reply:
The backend will fail anyway thanks to the implementation there...  (guess, you were more thoroughgoing ;-) )

But yes, it's slightly more polite to fail early (and creates less garbage). Added a simple check.